packages/contracts-bedrock/book/src/policies/versioning.md

```diff
@@ -124,27 +127,3 @@ contract could have the same semver, but different source/bytecode on the two br
 For example: if the current version of a contract is `1.1.1` and a minor bump is required (most common for a bug fix),
    then the fixed version should become `1.8.0` on `develop`. Then on the release branch is should become
    `1.2.0`.
-
-### Merging Back to Develop After Governance Approval
```

Comment:
This whole section was outdated and only applied when we were still including `-rc.n` on contract semvers.